src/utils/tokenExplorerApi.js

```diff
@@ -167,9 +174,9 @@ const parseBlockScout = async (json, address) => {
     return nft;
   });
   erc721s = await Promise.all(erc721s);
-
   const erc20s = json.result
     .filter((token) => token.type === 'ERC-20')
+    .sort((a, b) => b.balance - a.balance)
     .map((t) => {
       t.usd = tokenData[t.contractAddress.toLowerCase()]?.price || 0;
       t.totalUSD = parseFloat(+t.balance / 10 ** +t.decimals) * +t.usd;
```

Comment:
In keeping with the declarative, non-mutable ethos from the style guide, it would be better to return an object using object destructing syntax (ex. return {usd, totalUSD,}).